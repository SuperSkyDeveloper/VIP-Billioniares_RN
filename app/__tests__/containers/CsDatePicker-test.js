import React from 'react';
import renderer from 'react-test-renderer';
import { render, screen, fireEvent } from '@testing-library/react-native';
import CsDatePicker from '../../containers/CsDatePicker';

test('renders correctly', () => {
  const tree = renderer.create(<CsDatePicker />).toJSON();
  expect(tree).toMatchSnapshot();
});