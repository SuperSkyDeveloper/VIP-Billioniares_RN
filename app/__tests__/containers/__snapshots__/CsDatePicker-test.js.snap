// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "borderBottomColor": "grey",
        "borderBottomWidth": 0.5,
        "flexDirection": "row",
        "marginBottom": 10,
      },
      undefined,
    ]
  }
/>
`;
